miro_preprocessor/xml_to_json_run_task/src/xml_to_json_run_task.py

```diff
@@ -0,0 +1,61 @@
+# -*- encoding: utf-8 -*-
+
+"""
+Lambda to run the Miro XML to JSON task when receiving an S3 event.
```

Comment:
minor: I’d rephrase this to make it clear "run the task" means a specific AWS API:

> Lambda to call the RunTask API for the Miro XML-to-JSON ECS task when a file in S3 is updated.